odw/README.md

```diff
@@ -1,68 +1,215 @@
+
+## 🏗️ Tech Stack
+
+| Category             | Technologies                                        |
+| -------------------- | --------------------------------------------------- |
+| **Frontend**         | Streamlit, Python 3.12+                            |
+| **Backend**          | Moose, Python, FastAPI                             |
+| **Database**         | ClickHouse (analytical), PostgreSQL (temporal)     |
+| **Message Queue**    | Redpanda (Kafka-compatible)                        |
+| **Workflow Engine**  | Temporal                                           |
+| **Caching**          | Redis                                              |
+| **Data Processing**  | Kafka Python, ClickHouse Connect                   |
+| **Infrastructure**   | Docker, Docker Compose                             |
+| **Build Tool**       | Python setuptools, pip                             |
+
+## 📁 Project Structure
+
+```
+area-code/
+├── odw/                      # Operational Data Warehouse
+│   ├── services/            # Backend services
+│   │   ├── data-warehouse/  # Main Moose data warehouse service
+│   │   │   ├── app/
+│   │   │   │   ├── apis/    # REST API endpoints for data consumption
+│   │   │   │   ├── blobs/   # Blob data extraction workflows
+│   │   │   │   ├── events/  # Events data extraction workflows
+│   │   │   │   ├── ingest/  # Data models and stream transformations
+│   │   │   │   ├── logs/    # Log data extraction workflows
+│   │   │   │   └── views/   # Materialized views for analytics
+│   │   │   ├── docs/        # Documentation and walkthrough guides
+│   │   │   ├── setup.sh     # Setup and management script
+│   │   │   ├── moose.config.toml # Moose framework configuration
+│   │   │   └── requirements.txt # Python dependencies
+│   │   └── connectors/      # External data source connectors
+│   │       └── src/
+│   │           ├── blob_connector.py    # Blob storage data connector
+│   │           ├── events_connector.py  # Events data connector
+│   │           ├── logs_connector.py    # Logs data connector
+│   │           └── connector_factory.py # Connector factory pattern
+│   └── apps/
+│       └── dw-frontend/     # Streamlit web application
+│           ├── pages/       # Frontend page components
+│           ├── utils/       # Frontend utility functions
+│           └── main.py      # Streamlit application entry point
+```
+
+## 🔧 Services Overview
+
+### Frontend (dw-frontend)
+
+- Modern Streamlit web application with real-time data visualization
+- Interactive dashboards for analytics and monitoring
+- Real-time status monitoring and queue management
+- Responsive design with custom styling and tooltips
+
+### Data Warehouse Service
+
+- [Moose analytical APIs + ClickHouse](https://docs.fiveonefour.com/moose/building/consumption-apis) with automatic OpenAPI/Swagger docs
+- [Moose streaming Ingest Pipelines + Redpanda](https://docs.fiveonefour.com/moose/building/ingestion) for real-time data processing
```

Comment:
Should we also add something like:

[Moose workflows + Temporal](https://docs.fiveonefour.com/moose/building/workflows) for durable long-running data synchronization

Reply:
added